bundles/pom.xml

```diff
@@ -100,9 +100,10 @@
     <module>org.openhab.binding.fsinternetradio</module>
     <module>org.openhab.binding.ftpupload</module>
     <module>org.openhab.binding.gardena</module>
-    <module>org.openhab.binding.goecharger</module>
```

Comment:
This shouldn't have been moved...

Reply:
but it's at the wrong alphabetical order

Reply:
changed reverted

Reply:
changed reverted